packages/core/src/hydration/event_replay.ts

```diff
@@ -58,11 +57,18 @@ export function withEventReplay(): Provider[] {
     {
       provide: ENVIRONMENT_INITIALIZER,
       useValue: () => {
-        setDisableEventReplayImpl((el: RElement) => {
-          if (el.hasAttribute(JSACTION_ATTRIBUTE)) {
+        setDisableEventReplayImpl((rEl, eventName, listenerFn) => {
+          if (rEl.hasAttribute(JSACTION_ATTRIBUTE)) {
+            const el = unwrapRNode(rEl) as Element;
```

Comment:
nit: we can add types to the callback function and remove the `unwrapRNode` call, since we already have a DOM element available (`RElement` is an internal type for a DOM element).
```suggestion
        setDisableEventReplayImpl((el: Element, eventName: string, listenerFn: VoidFunction) => {
          if (el.hasAttribute(JSACTION_ATTRIBUTE)) {
```

Reply:
done